kornia/augmentation/random_generator3d.py

```diff
@@ -101,19 +101,21 @@ def random_affine_generator3d(
 
     if translate is not None:
         assert translate.shape == torch.Size([3]), f"'translate' must be the shape of (2). Got {translate.shape}."
-        max_dx: torch.Tensor = translate[0] * depth
-        max_dy: torch.Tensor = translate[1] * width
-        max_dz: torch.Tensor = translate[2] * height
+        max_dz: torch.Tensor = translate[0] * depth
+        max_dy: torch.Tensor = translate[1] * height
+        max_dx: torch.Tensor = translate[2] * width
+        # translations should be in x,y,z
         translations = torch.stack([
             _adapted_uniform((batch_size,), -max_dx, max_dx, same_on_batch),
             _adapted_uniform((batch_size,), -max_dy, max_dy, same_on_batch),
             _adapted_uniform((batch_size,), -max_dz, max_dz, same_on_batch)
-        ], dim=-1)
+        ], dim=1)
     else:
         translations = torch.zeros(batch_size, 3)
 
+    # center should be in x,y,z
     center: torch.Tensor = torch.tensor(
-        [depth, width, height], dtype=torch.float32).view(1, 3) / 2. - 0.5
+        [width, height, depth], dtype=torch.float32).view(1, 3) / 2. - 0.5
```

Comment:
is there any test that reflects this issue ?

Reply:
There is no test for 3D Affine transform so far. I'll add some later, though due to #670 I guess soon all of them are gona get refactored.